network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_breadcrumbs.html

```diff
@@ -0,0 +1,6 @@
+{% load wagtailcore_tags %}
+
+<div class="tw-text-sm tw-text-gray-40">
+    <a class="tw-text-gray-40" href="{% pageurl page %}">{{ page.title }}</a>
+    <span> › </span>
```

Comment:
Let's change this so it's more a11y friendly. 

Using HTML entity `&gt;` for the symbol > makes sure the separator is properly encoded and can be read by screen readers etc. The surrounding spacing would be better if it's done via CSS than the actual space character.

One thing I'm unsure because the lack of attached mockup is that using Tailwind classes to style as well as `&gt;` does give a different look (screen shot no.1) than what's currently implemented in this PR (screen shot no.2). We might need to change the font styling/type so the separator look the way we wanted.

```suggestion
    <span aria-hidden="true" class="tw-mx-1">&gt;</span>
```

Screencap no.1

<img width="61" alt="image" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/2896608/09f32b6a-4f4a-406b-b6e2-bf04feef5956">

Screencap no.2

<img width="55" alt="image" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/2896608/a4d2141e-5c54-4fcf-a684-8140b8e3f641">

Reply:
Thanks @mmmavis, those are awesome suggestions! Accesibility is definitely not my strongest suit, I appreciate the input.

One thing to note is that the char is not the "greater than" quotation mark (>) but rather the "single right-pointing angle quotation mark" (`›`) -- see https://www.compart.com/en/unicode/U+203A.

The HTML entity seems to be `&rsaquo;`.